### DISCUSSION TITLE
The SubscriptionId field is required

### DISCUSSION THREAD

**Discussion by timvdh** [shibayan/appservice-acmebot#449] [Q&A] [answered]
For some reason the appservice-acmebot bot stopped working and the function overview states: "Failed to load functions" (I am not aware that I changed anything). I thought this is a good time to upgrade from v3 to v4, so I changed 

FUNCTIONS_EXTENSION_VERSION=~4
WEBSITE_RUN_FROM_PACKAGE=https://stacmebotprod.blob.core.windows.net/appservice-acmebot/v4/latest.zip

in env. The func still can't load, but now overview also states:

Microsoft.Extensions.Options: DataAnnotation validation failed for 'AcmebotOptions' members: 'SubscriptionId' with the error: 'The SubscriptionId field is required.'.

I tried adding Acmebot:SubscriptionId in env (Acmebot:AzureDns:SubscriptionId already existed) but same result.

**Comment by shibayan** [comment, marked as answer]
That error occurs when `Acmebot:SubscriptionId` is not set correctly, so please make sure it is saved again. Then, if necessary, restart the system to see if the error has been resolved.

**Reply by timvdh** [reply]
Yes, functions worked again the other day, I didn't even have to do anything.
